toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -352,24 +392,164 @@ func findPartition(mountIdType imagecustomizerapi.MountIdentifierType, mountId s
 	return partition, partitionIndex, nil
 }
 
+func convertDevToUuid(partitions []diskutils.PartitionInfo, buildDir string) (string, error) {
```

Comment:
The `extractVerityRootPartUUID` function is returning the partition's UUID. So, the for-loop inside the `convertDevToUuid` function is pointless.